### ISSUE TITLE
Wrong output size when generating a SVG (on Mac)

### ISSUE THREAD

**Issue by Atcold** [Jonathan-LeRoux/IguanaTex#39]
IguanaTeX output (top) is 87% × 97% while LaTeXiT's (bottom) is 100% × 100%.

<img width="887" alt="Screenshot 2022-10-13 at 15 36 53" src="https://user-images.githubusercontent.com/2119355/195694540-ec42a3dc-8f42-4fec-8494-facd39c88483.png">

**Comment by Atcold** [comment]
I scaled it up to 100% × 100% manually.
The two outputs are now identical if not for the extended bounding box for IguanaTeX (perhaps that's what causes the wrong output size).

<img width="589" alt="image" src="https://user-images.githubusercontent.com/2119355/195695955-749b4e4c-ffde-49b6-84e8-37eae5a8ab6c.png">

**Comment by Atcold** [comment]
Something's really wrong…

<img width="941" alt="image" src="https://user-images.githubusercontent.com/2119355/195698248-7e6cbf2d-5dc5-49d1-9281-91c6d9658c56.png">

**Comment by Jonathan-LeRoux** [comment]
Hi! Are you using version 1.60.1 or newer? There should no longer be a big margin around the equation when generating a "Bitmap" (which on Mac is actually a PDF...).
This unfortunately results in the equation appearing cropped on Windows machines (bounding box stays the same but the content inside expands; most likely a PPT bug), but I haven't found a fix yet, short of moving to PNGs for Mac as well. Windows users can always regenerate the display as a PNG, it will get back to the same appearance as on Mac.

For you last example with colors, was that a "vector graphics" display? Those are indeed SVGs. The scale does look very strange...
The "Bitmap" displays are PDFs, although under the hood PPT is storing them as EMFs (even on Mac!). IguanaTex shouldn't be converting them to SVG.

**Comment by Atcold** [comment]
Aaaaaaah!
PDF is not bitmap! LOL.

Everything's perfect now.
I'd heartwarmingly suggest to change "Bitmap" for "PDF".

**Comment by Jonathan-LeRoux** [comment]
I agree, "Bitmap" is not a good term, it is used because that's what it is under Windows (PNG). The PDF ("Bitmap") displays on Mac also cannot have their paths edited or individual components tweaked independently, as the "Vector" displays can, so they are closer to the PNGs on Windows in that regard.

I'll have to think about how to handle this. 
Easiest in terms of implementation would be to find a term that describes both PNG and PDF cases, but I can't see one. 
Another option would be to adapt the terminology on Windows vs Mac, but I don't know if the ribbon text can be made adaptive on Mac vs Windows, as it is handled outside of VBA in an XML file; that may require having two separate versions of the add-in.
Yet another option would be to implement PNG on Mac for "Bitmap" displays, and provide PDF as a "Vector" option, with the caveat that it has the crispness of vector graphics format but cannot be edited.

**Comment by Atcold** [comment]
So, let me summarise.
The wrong aspect ratio was happening on a Mac when choosing "verctorial" output (which gets converted to a group of shapes).
Choosing "bitmap" output (which is **not** bitmap, but it's vectorial PDF) the aspect ratio looks just fine.
I'd recommend changing the label on the plugin in order to avoid confusion for other users.

**Comment by Atcold** [comment]
> Vector graphics are digital art that is rendered by a computer using a mathematical formula. Raster images are made up of tiny pixels, making them resolution dependent and best used for creating photos.

See [reference](https://pavilion.dinfos.edu/Article/Article/2223089/vector-vs-raster-images-choosing-the-right-format/#:~:text=and%20appears%20smooth.-,Vector%20graphics%20are%20digital%20art%20that%20is%20rendered%20by%20a,best%20used%20for%20creating%20photos.).

Vector graphics does not have anything to do with editability.

I'd suggest having the user choosing between "SVG", "PDF", and "PNG".

A PDF is a container. It can contain both a raster and / or a vector image.
PowerPoint cannot edit PDFs. This does not mean PDFs are not editable in their individual components.

**Comment by Atcold** [comment]
Reopening this because the bug persists, although it no longer affects me (as in I'm perfectly fine with generating PDFs instead of SVGs).
Let me know if I can help debug this.

So, the reported size is most of the time 100% × 100%.
If you click and unclick *several* times, you finally get the real wrong aspect ratio.

A workaround is generating a PDF first and then converting it to SVG (although it says sometimes it's 112% × 100%).

![Screenshot 2022-10-17 at 12 51 28](https://user-images.githubusercontent.com/2119355/196237268-686a5e2e-29a7-4737-a4aa-547f431f3ffc.png)
![Screenshot 2022-10-17 at 12 51 44](https://user-images.githubusercontent.com/2119355/196237375-ec760625-ebe4-405f-b0d4-4376245ceb1e.png)
![Screenshot 2022-10-17 at 12 51 06](https://user-images.githubusercontent.com/2119355/196237395-70cce517-cfb5-44a3-a4b8-63a5b9a53a3b.png)
